Jenkinsfile-rf-pipeline

```diff
@@ -0,0 +1,308 @@
+
+                }
+
+                currentStage = "ci-pipeline-ostree-compose"
+                stage(currentStage) {
+                    // Set stage specific vars
+                    pipelineUtils.setStageEnvVars(currentStage)
+
+                    //Set our message topic, properties, and content
+                    (topic, messageProperties, messageContent) = pipelineUtils.setMessageFields("compose.running")
+
+                    // Send message org.centos.prod.ci.pipeline.compose.running on fedmsg
+                    sendMessage(messageProperties, messageContent)
+
+                    // Provision resources
+                    pipelineUtils.provisionResources(currentStage)
+
+                    // Stage resources - ostree compose
+                    pipelineUtils.setupStage(current_stage, 'fedora-atomic-key')
+
+                    // Rsync Data
+                    pipelineUtils.rsyncData(currentStage)
+
+                    def ostree_props = "${env.ORIGIN_WORKSPACE}/logs/ostree.props"
+                    def ostree_props_groovy = "${env.ORIGIN_WORKSPACE}/ostree.props.groovy"
+                    pipelineUtils.convertProps(ostree_props, ostree_props_groovy)
+                    load(ostree_props_groovy)
+
+                    // Teardown resource
+                    pipelineUtils.teardownResources(currentStage)
+
+                    // Set our message topic, properties, and content
+                    (topic, messageProperties, messageContent) = pipelineUtils.setMessageFields("package.complete")
+
+                    // Send message org.centos.prod.ci.pipeline.package.complete on fedmsg
+                    sendMessage(messageProperties, messageContent)
+
+                    checkLastImage(currentStage)
+                }
+
+                currentStage = "ci-pipeline-ostree-image-compose"
+                stage(currentStage) {
+                    // Check if a new ostree image compose is needed
+                    if (fileExists("${env.WORKSPACE}/NeedNewImage.txt") || ("${env.GENERATE_IMAGE}" == "true")) {
+                        // Set stage specific vars
+                        pipelineUtils.setStageEnvVars(currentStage)
+
+                        // Set our message topic, properties, and content
+                        (topic, messageProperties, messageContent) = pipelineUtils.setMessageFields("image.running")
+
+                        // Send message org.centos.prod.ci.pipeline.image.running on fedmsg
+                        sendMessage(messageProperties, messageContent)
+
+                        // Provision resources
+                        pipelineUtils.provisionResources(currentStage)
+
+                        // Stage resources - ostree image compose
+                        pipelineUtils.setupStage(current_stage, 'fedora-atomic-key')
+
+                        // Rsync Data
+                        pipelineUtils.rsyncData(currentStage)
```

Comment:
You changed the name of the method so this needs to be updated

Reply:
rsyncData() didn't change name. The method rsyncResults was renamed to runTaskAndReturnLogs(). 
This wouldn't need to be updated.

Reply:
Then why do we need runTaskAndReturnLogs() for?

Reply:
kk